### ISSUE TITLE
Bijector Checklist and Progress

### ISSUE THREAD

**Issue by lockwo** [lockwo/distreqx#5]
- [x] bijector
- [x] block
- [x] chain
- [x] diag linear
- [ ] diag plus low rank linear
- [ ] Gumbel cdf
- [ ] inverse
- [ ] lambda
- [x] linear
- [ ] lower upper triangular affine
- [ ] masked coupling
- [ ] rational quadratic spline
- [x] scalar affine
- [x] shift
- [ ] sigmoid
- [ ] split coupling
- [x] tanh
- [ ] triangular linear
- [ ] unconstrained affine

**Comment by gvcallen** [comment]
Hi @lockwo similar to the distribution's checklist here are the higher priority bijectors for my downstream libraries split into two groups:

Foundational:
- Inverse
- Identity
- Leafwise

Utilities:
- Indexed
- Split
- Restructure
- Permute
- Reshape

Full disclosure that I did use Claude for some of the recent changes and then reviewed the code manually after.

Please let me know if there is anything I can help with.

**Comment by lockwo** [comment]
Hey I think the biggest way to help speed stuff up is here https://github.com/lockwo/distreqx/pull/67#issuecomment-4932217545. I have a bunch of PRs myself recently shoring up the style and format, so check out the new changes (e.g. jax.tree, docstring stuff, fields, etc.). The ones that are new do take a little longer to review (by new, I mean different from/not in distrax), but I would like to support people building on distreqx, so I will do my best to prioritize the following 

TruncatedNormal
ImproperUniform (DONE)
Joint (if you want that faster you can also split up the PR into a new one so deterministic isn't part of it)
Inverse (DONE)
Identity (DONE)
Leafwise

**Comment by gvcallen** [comment]
@lockwo awesome thanks! Yes I really feel the distreqx way is the right way (no offence distrax) hence why I chose it, but there are some gaps making it a bit challenging for complex use-cases!

FYI, I made a (very early-stage) normalizing flow library, [fleqx](https://github.com/gvcallen/fleqx), using Claude to translate stuff from FlowJAX with some additional features. For now I am using it for distreqx-native coupling flows for my PhD and it works great, but I still need to review the code hence why I have not shared it elsewhere
